### PR TITLE
feat(flutter): isar integration

### PR DESCRIPTION
**Pull request by denrase**
Add isar integration docs for Flutter/Dart

Relates to https://github.com/getsentry/sentry-dart/issues/1501

### CODE REVIEW

**Comment by denrase** [comment]
@buenaflor FYI

**Review by vivianyentran — APPROVED** [review]

**Comment by vivianyentran** [comment]
LGTM, thanks for adding this doc! @buenaflor can you also take a look?

File: docs/platforms/flutter/integrations/isar-instrumentation.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: Isar Database Instrumentation
+description: "Learn more about the Sentry Isar Database Instrumentation for the Flutter SDK."
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.dart.isar
```
Comment (buenaflor) [inline docs/platforms/flutter/integrations/isar-instrumentation.mdx:6]: I think this should be `sentry.dart.flutter` ?

Comment (buenaflor) [reply docs/platforms/flutter/integrations/isar-instrumentation.mdx:6]: The other docs are prolly incorrect that say `sentry.dart.drift` for example

**Review by buenaflor — APPROVED** [review]
left a comment but looks good!

**Comment by denrase** [comment]
@vivianyentran Oversaw that this PR is still open. Would you be so kind and run Vercel so we can merge? 🙇

**Comment by denrase** [comment]
@buenaflor This was closed as I could not run vercel, can you revert this?

**Comment by buenaflor** [comment]
maybe @lizokm can re-run the vercel deployment pls

**Comment by denrase** [comment]
@buenaflor Wen we get out of sync with the main repo we always need an additional vercel run. So any push to the main repo blocks us potentially here. 😅  Is there an option on the PR to automatically merge when all required checks are green?

**Comment by vivianyentran** [comment]
Just reran the vercel deployment to merge this!

**Pull request merged**